app/utils/onboard.ts

```diff
@@ -12,7 +12,11 @@ const injected = injectedModule();
 
 // web3Onboard modules
 
-const walletConnect = walletConnectModule();
+const walletConnectOptions: WalletConnectOptions = {
+  projectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID as string,
```

Comment:
I think this might need an `or empty string`
```suggestion
  projectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID as string || "",
```